daemons/controld/controld_schedulerd.c

```diff
@@ -31,11 +31,11 @@ static pcmk_ipc_api_t *schedulerd_api = NULL;
  * \brief Close any scheduler connection and free associated memory
  */
 void
-pe_subsystem_free(void)
+controld_shutdown_ipc(void)
```

Comment:
The name should indicate that this is specifically for the scheduler IPC -- the controller has its own IPC, and also connects to executor and fencer IPC

Reply:
Fixed with some better names, hopefully.